ros/.env.example

```
@@ -0,0 +1,5 @@
+ROS_DB_USER="postgres"
+ROS_DB_PASS="postgres"
+ROS_DB_HOST="localhost"
+ROS_DB_PORT="15432"
+ROS_DB_NAME="postgres"
```

```suggestion
ROS_DB_NAME="postgres"

INVENTORY_HOST="localhost"
INVENTORY_PORT="8081"

INSIGHTS_KAFKA_HOST="localhost"
INSIGHTS_KAFKA_PORT="29092"
```

We should also add host-inventory and Kafka config here.